src/studio/src/designer/frontend/shared/features/dataModelling/functions/findPreferredMetadataOption.ts

```diff
@@ -0,0 +1,10 @@
+import { IMetadataOption } from './types';
+
+function findPreferredMetadataOption(metadataOptions: IMetadataOption[], preferred?: string) {
```

Comment:
Any reason for defining each function in a separate file and not a collection of utils grouped together? 
For instance: `altinn-studio\src\studio\src\designer\frontend\packages\schema-editor\src\utils.ts`

Reply:
Do we prefer large files instead of shorter files whit fewer dependencies?
If I'm looking for a util, in my opinion, it is easier to search for the file matching the function than remembering which util-file it was written inside of.

Reply:
I feel like it would be better to follow the established pattern of gathering utils functions in a single file just to get consistency when navigating the project.

I do see your point though 👍

Reply:
This isn't really a util either, it is only used by the Data-Modellng as a function that could live inside the tsx. However, to keep file size down, I did extract it into another file.